protos/core.proto

```diff
@@ -36,12 +36,17 @@ message RedshiftOptions {
   string sort_style = 4;
 }
 
+message GraphErrors {
+  repeated CompilationError compilation_errors = 1;
+  repeated ValidationError validation_errors = 2;
+}
+
 message ValidationError {
-  string file_name = 1;
+  string node_name = 1;
   string message = 2;
 }
 
-message CompileError {
+message CompilationError {
   string file_name = 1;
```

Comment:
should this be node_name too?

Reply:
> should this be node_name too?
no, this is filename - link to file with error!